Content.Server/Chemistry/ReagentEffects/WearableReaction.cs

```diff
@@ -0,0 +1,31 @@
+using Content.Shared.Inventory;
+using Content.Shared.Chemistry.Reagent;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Chemistry.ReagentEffects
```

Comment:
Please switch to [file-scoped namespaces](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#file-layout).